### PR TITLE
Added a Matlab function

### PR DESCRIPTION
**Pull request by mikelgg93**
For those who do not use R but Matlab. ;)

### CODE REVIEW

File: Matlab/color_Mett.m [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    map= validatestring(map,validColorMaps);
+    if verbosity; fprintf(['color_Met>> Retrieving colormap ', upper(char(map)), '... \n']);end
+    
+    % Anonymous function to convert from HEX to RGB, credits to Chad Greene
+    % See hex2rgb.
+    rgb = @(hex)(reshape(sscanf(strrep(hex, '#','').','%2x'),3,[]).'/255);
+    
+    % This color palet was originally created for R by Blake Robert Mills.
+    % See https://github.com/BlakeRMills/MetBrewer/tree/main
+   function output = color_Mett(map, minColor, verbosity)
```
Comment (git-thor) [inline Matlab/color_Mett.m]: You have broken your script by accident, defining the function and documentation again, starting here in line 35

File: Matlab/color_Mett.m [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    c.Troy = {'#421401'; '#6c1d0e'; '#8b3a2b'; '#c27668'; '#7ba0b4'; '#44728c'; '#235070'; '#0a2d46'}; 
+    c.Tsimshian ={'#582310'; '#aa361d'; '#82c45f'; '#318f49'; '#0cb4bb'; '#2673a3'; '#473d7d'};
+    c.VanGogh1 = {'#2c2d54'; '#434475'; '#6b6ca3'; '#969bc7'; '#87bcbd'; '#89ab7c'; '#6f9954'}; 
+    c.VanGogh2 = {'#bd3106'; '#d9700e'; '#e9a00e'; '#eebe04'; '#5b7314'; '#c3d6ce'; '#89a6bb'; '#454b87'}; 
+    c.VanGogh3 = {'#e7e5cc'; '#c2d6a4'; '#9cc184'; '#669d62'; '#447243'; '#1f5b25'; '#1e3d14'; '#192813'};
+    c.Veronese = {'#67322e'; '#99610a'; '#c38f16'; '#6e948c'; '#2c6b67'; '#175449'; '#122c43'};
+    c.Wissing= {'#4b1d0d'; '#7c291e'; '#ba7233'; '#3a4421'; '#2d5380'};
+    
+    % Calculate how many steps in between are needed
+    if size(c.([map]),1) < minColor
```
Comment (git-thor) [inline Matlab/color_Mett.m]: Matlab does not like `c` without an initial definition as struct in before.

File: Matlab/color_Mett.m [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    "Tsimshian","Thomas", "Tiepolo", "Troy", "VanGogh1", "VanGogh2", "VanGogh3","Veronese","Wissing"];
+    map= validatestring(map,validColorMaps);
+    if verbosity; fprintf(['color_Met>> Retrieving colormap ', upper(char(map)), '... \n']);end
+    
+    % Anonymous function to convert from HEX to RGB, credits to Chad Greene
+    % See hex2rgb.
+    rgb = @(hex)(reshape(sscanf(strrep(hex, '#','').','%2x'),3,[]).'/255);
+    
+    % This color palet was originally created for R by Blake Robert Mills.
+    % See https://github.com/BlakeRMills/MetBrewer/tree/main
```
Comment (git-thor) [inline Matlab/color_Mett.m]: Proposing the definition of the color struct as `c = struct();`

File: Matlab/color_Mett.m (hunk 1 above)
Comment (mikelgg93) [reply Matlab/color_Mett.m]: I didnot had time the other day to check it, but seems like the commit got corrupted. Should be fixed now.

File: Matlab/color_Mett.m (hunk 2 above)
Comment (mikelgg93) [reply Matlab/color_Mett.m]: now it is initialised before

**Comment by mikelgg93** [comment]
I renamed the function to match the original one.
I also added a way to see all colour palettes at once.  See MetBrewer('Demo')

**Pull request closed without merging**